library/ajax/drug_autocomplete/search.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
Remove the following files above:
library/ajax/drug_autocomplete/js/jquery-1.9.1.min.js
library/ajax/drug_autocomplete/js/jquery-ui.min.css
library/ajax/drug_autocomplete/js/jquery-ui.min.js
library/ajax/drug_autocomplete/js/jquery-ui.min.js